### ISSUE TITLE
Import script attemps to copy using Windows file path on Linux

### ISSUE THREAD

**Issue by gstorer** [xavier150/Blender-For-UnrealEngine-Addons#85]
When exporting I am getting the follow error. This is happening because you are using a hard coded windows path:
https://github.com/xavier150/Blender-For-UnrealEngine-Addons/blob/7731e29d7e8100f6eb2237b48c6ac38d034ed9af/blender-for-unrealengine/bfu_write_text.py#L500
This line should be 
`  source = os.path.join(bpy.utils.user_resource('SCRIPTS', os.path.join("addons", "blender-for-unrealengine", "import")), "asset_import_script.py") `

```
Error: Python: Traceback (most recent call last):
  File "/home/g/.config/blender/2.91/scripts/addons/blender-for-unrealengine/bfu_ui.py", line 2214, in execute
    bfu_write_text.WriteAllTextFiles()
  File "/home/g/.config/blender/2.91/scripts/addons/blender-for-unrealengine/bfu_write_text.py", line 503, in WriteAllTextFiles
    copyfile(source, destination)
  File "/snap/blender/65/2.91/python/lib/python3.7/shutil.py", line 120, in copyfile
    with open(src, 'rb') as fsrc:
FileNotFoundError: [Errno 2] No such file or directory: '/home/g/.config/blender/2.91/scripts/addons\\blender-for-unrealengine\\import/asset_import_script.py'

location: <unknown location>:-1
```

I am using release 0.2.8.3 with Blender 2.91.2 and Unreal 4.26.1 on Ubuntu 20.04.2 LTS.

**Comment by xavier150** [comment]
Hello @gstorer
Thanks for the report! I fixed and updated the source, I will soon update the addon.

**Comment by gstorer** [comment]
The dev branch version has fixed this for me now.

**Comment by xavier150** [comment]
Hey @gstorer I published the new [update 0.2.8](https://github.com/xavier150/Blender-For-UnrealEngine-Addons/releases/tag/v0.2.8) 
So I close this issue :)
